spotbugs/src/main/java/edu/umd/cs/findbugs/ba/npe/IsNullValueFrame.java

```diff
@@ -250,8 +250,14 @@ public boolean sameAs(Frame<IsNullValue> other) {
             return false;
         }
         IsNullValueFrame o2 = (IsNullValueFrame) other;
-        return Objects.equals(decision, o2.decision)
```

Comment:
Why undo this?

Reply:
I just keep it same with `release-3.1`. Sorry I didn't notice that the change 84859d131c1c25346ab8f8ec83d1d674a7eced0f exists only in `master` branch.